horace_core/sqw/@sqw/private/unary_op_manager.m

```diff
@@ -14,10 +14,8 @@
 w = w1;
 for i=1:numel(w1)
     if is_sqw_type(w1(i))
```

Comment:
does the `else` here need updating too?

Reply:
I don't think so, the `else` is for dnd data, so no pixels